src/chrome/content/rules/163.com-mixedcontent.xml

```diff
@@ -1,23 +1,125 @@
 <!--
 	For rules not causing  MCB, see 163.com.xml.
-
 -->
-<ruleset name="163.com (MCB)" platform="mixedcontent">
+
+<ruleset name="163.com (MCB)">
 
 	<target host="1.163.com" />
+	
+		<exclusion pattern="^http://1.163.com/$" />
+		
+		<rule from="^http://1\.163\.com/favicon\.ico/"
+			to="https://1.163.com/favicon.ico/" />
+			<test url="http://1.163.com/favicon.ico/" />
+		
+		<rule from="^http://1\.163\.com/(appguide|goods|hd|kpimission|user)/(.+)\.do"
+			to="https://1.163.com/$1/$2.do" />
+			<test url="http://1.163.com/appguide/getconfig.do" />
```

Comment:
You are including every *.do, so you also need to test more than one *.do for each path.